examples/react/getting-started/package.json

```diff
@@ -7,7 +7,7 @@
     "start": "BABEL_ENV=parcel parcel index.html products.html --port 3000"
   },
   "dependencies": {
-    "algoliasearch": "5.1.1",
+    "algoliasearch": "5.4.0",
```

Comment:
can you leave this at 5.1.1 for now? we can update all at once in the examples in the whole repo